our-work/collaborations/crunch.yml

```diff
@@ -0,0 +1,15 @@
+title: The Crunch Group
+id: crunch-group
+description: The CRUNCH group is a research team of the Division of Applied Mathematics working at the interface of Computational Mathematics + Machine Learning + X, where X may be problems in biology, geophysics, soft matter, functional materials, physical chemistry, or fluid and solid mechanics. Their emphasis is on Scientific Machine Learning and Physics-Informed Neural Networks (PINNs) that the CRUNCH group pioneered. 
+tags:
+  - active
+  - applied-math
```

Comment:
I'm also wondering if we add a tag for `system-administration` or some other tag that relates/identifies to the kind of work you are doing for the crunch group.